examples/02_Parameter Estimation.ipynb

```diff
@@ -13,38 +13,54 @@
+    "Parameter estimation is used to tune the parameters of a general model so that its behavior matches that of a specific system. Parameter Estimation can be considered more of an art than a science. For example, the parameters of a battery model can be tuned to configure the model to more accurately describe the behavior of a specific battery.\n",
     "\n",
-    "Generally, parameter estimation is done by tuning the parameters of the model so that simulation (see 1. Simulation) best matches the behavior observed in some available data. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationall expensive. Fortunately, in this case we have a relatively simple model.\n",
+    "Generally, parameter estimation is done by tuning the parameters of the model so that the simulation (see __[01 Simulation](01_Simulation.ipynb)__) best matches the behavior observed in some available data. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationally expensive. Fortunately, in this case we have a relatively simple model.\n",
     "\n",
-    "In ProgPy, this is done using the progpy.PrognosticsModel.estimate_params() method. This method takes input and output data from one or more runs, and uses scipy.optimize.minimize function to estimate the parameters of the model. For more information, refer to our Documentation [here](https://nasa.github.io/progpy/prog_models_guide.html#parameter-estimation)\n",
+    "In ProgPy, this is done using the `progpy.PrognosticsModel.estimate_params()` method. This method takes input and output data from one or more runs, and uses `scipy.optimize.minimize` function to estimate the parameters of the model. For more information, refer to the documentation [here](https://nasa.github.io/progpy/prog_models_guide.html#parameter-estimation).\n",
     "\n",
     "A few definitions:\n",
+    "\n",
     "* __`keys`__ `(list[str])`: Parameter keys to optimize\n",
     "* __`times`__ `(list[float])`: Array of times for each run\n",
     "* __`inputs`__ `(list[InputContainer])`: Array of input containers where inputs[x] corresponds to times[x]\n",
     "* __`outputs`__ `(list[OutputContainer])`: Array of output containers where outputs[x] corresponds to times[x]\n",
-    "* __`method`__ `(str, optional)`: Optimization method- see scipy.optimize.minimize for options\n",
+    "* __`method`__ `(str, optional)`: Optimization method. See `scipy.optimize.minimize` for options\n",
     "* __`tol`__ `(int, optional)`: Tolerance for termination. Depending on the provided minimization method, specifying tolerance sets solver-specific options to tol\n",
     "* __`error_method`__ `(str, optional)`: Method to use in calculating error. See calc_error for options\n",
```

Comment:
Do we want to link to calc_error documentation here?

Reply:
Yup - linked in `error_method`